ndonnx/_funcs.py

```diff
@@ -279,7 +280,6 @@ def isdtype(dtype, kind):
 def result_type(
     *objects: dtypes.CoreType | dtypes.StructType | Array,
 ) -> dtypes.CoreType | dtypes.StructType:
-    """Find the common data type for the provided objects."""
```

Comment:
Was that deliberate?

Reply:
Yes, it is overwritten https://github.com/Quantco/ndonnx/blob/main/ndonnx/_funcs.py#L1038-L1041.